srcDLL/Globals.cpp

```diff
@@ -7,5 +7,5 @@
 bool modulesRunning = false;
 
 VolList volList;
-ConsoleModuleLoader consoleModLoader(FindModuleDirectory());
+ConsoleModuleLoader consoleModLoader({FindModuleDirectory()});
```

Comment:
What does it mean to add the curly braces around FindModuleDirectory here?

Reply:
The constructor now takes `std::vector` rather than `std::string`. The braces cause a vector to be constructed with that one element.

It's a bit of a temporary hack until we get command line parsing code that will return a vector of module names instead of just a single one.